stdlib/Distributed/test/distributed_exec.jl

```diff
@@ -803,6 +803,7 @@ fetch(v15406)
 remotecall_wait(fetch, id_other, v15406)
 
 # Test various forms of remotecall* invocations
+println("\nTesting various forms of remotecall invocations...")
```

Comment:
Tests should be silent when successful. Use a testset for groupings.

Reply:
Yes. Should be changed all over in this test script.

Reply:
done in https://github.com/JuliaLang/julia/pull/32290/commits/4a92628567535dd53881d384706499963aa76f7b. Using testsets for groupings should probably be done in a separate PR as that will result in extensive changes to the test script and unrelated to this PR.